content/blog/esc-ga-announcement/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Announcing the General Availability of Pulumi ESC - Welcome to the new era of Secrets Management"
+allow_long_title: true
+date: 2024-09-18T00:00:00-03:00
+draft: false
+meta_desc: "Pulumi ESC is the centralized secrets and configuration management platform that allwo you to easily access, share, and manage securely on any cloud, using your favorite programming languages."
```

Comment:
I had suggested to @thtmnisamnstr some tweaks to this for the messaging - I don't want to do something different here than elsewhere - but I do think we shoudl consistenly do something stronger than this.

My suggestions was this - but he may have a more polished replacement:

```
Easily manage and tame secrets and configuration complexity across all of your cloud infrastructure and application environments.
```

Reply:
Updating this in the next commit